test/project_types/script/layers/application/extension_points_test.rb

```diff
@@ -106,7 +106,7 @@
       let(:scripts_beta_extension_points) { true }
 
       it "returns a list of all languages implemented by all extension points" do
-        assert_equal ["assemblyscript", "rust", "tinygo"], subject
+        assert_equal ["assemblyscript", "rust", "other", "tinygo"], subject
```

Comment:
The ordering here is a bit non-ideal, but I think this is fine. Let me know if you disagree.
Ideally, "other" would always end up as the last language in lists like these.

However, because the adding of "other" is tied to a specific type's languages, when we look at `.all_languages`, we: 
- look at all types,
- for each type, get all languages <- other is added here
- end up with duplicates, for many languages, that we `uniq` out of.

This means that if any language isn't part of the first type, that type's languages will be after "other".